code/game/machinery/spaceheater.dm

```diff
@@ -324,10 +324,11 @@
 	/// How quickly it delivers heat to the reagents. In watts per joule of the thermal energy difference of the reagent from the temperature difference of the current and target temperatures.
 	var/beaker_conduction_power = 0.1
 	/// The subsystem we're being processed by.
-	var/datum/controller/subsystem/processing/our_subsystem
+	var/static/datum/controller/subsystem/processing/our_subsystem = null
```

Comment:
invalid, breaks on MC recreation. don't do this just keep it as is

Reply:
Ohhh. Yup i'll put it back